modin/core/execution/ray/implementations/pandas_on_ray/io/io.py

```diff
@@ -165,7 +165,8 @@ def to_csv(cls, qc, **kwargs):
             Parameters for ``pandas.to_csv(**kwargs)``.
         """
         if not cls._to_csv_check_support(kwargs):
-            return RayIO.to_csv(qc, **kwargs)
+            RayIO.to_csv(qc, **kwargs)
+            return
```

Comment:
I'm not sure I like this change; original was one line shorter, and this form doesn't look cleaner to me - what warning did this fix?

Reply:
@vnlitvinov

Reply:
@vnlitvinov So what are we going to do about it? Disable checks of this type in `CodeQL` or will we write code in this style?